cookbook/api_examples/async_api_example.py

```diff
@@ -0,0 +1,43 @@
+"""An asynchronous FastAPI app integrated with a multi-chain prompt for recommending books on a topic 
```

Comment:
line too long. Make sure it fits cleanly within the ruler with a period for the first line. Then include a newline between that and any additional text.